caniusepython3/__main__.py

```diff
@@ -68,34 +68,47 @@ def projects_from_cli(args):
 
 def message(blockers):
     """Create a sequence of key messages based on what is blocking."""
+    encoding = getattr(sys.stdout, 'encoding', '')
+    if encoding:
+        encoding = encoding.lower()
+
     if not blockers:
-        encoding = getattr(sys.stdout, 'encoding', '')
-        if encoding:
-            encoding = encoding.lower()
         if encoding == 'utf-8':
             # party hat
             flair = "\U0001F389  "
         else:
             flair = ''
-        return [flair +
-                'You have 0 projects blocking you from using Python 3!']
+
+        return [flair + 'You have no projects known to block you from using Python 3!']
+
     flattened_blockers = set()
     for blocker_reasons in blockers:
         for blocker in blocker_reasons:
             flattened_blockers.add(blocker)
-    need = 'You need {0} project{1} to transition to Python 3.'
-    formatted_need = need.format(len(flattened_blockers),
-                      's' if len(flattened_blockers) != 1 else '')
+
+    if encoding == 'utf-8':
```

Comment:
Should probably change this to not be so static to UTF-8 as it isn't actually dependent on that encoding, just _an_ encoding that can handle it.